src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -51,141 +52,1161 @@
     static boolean isDenaliOrLater = false;
 
```

Comment:
add a negative test for DW to make sure it throws the proper error

Reply:
Well...we can't. Geometry/Geography are not supported datatypes against DW. We wouldn't be able to test it because we can't create a table with those column types.

Reply:
that's what I mean, add a negative test that runs on DW and check that it will fail with the appropriate unsupported error msg

Reply:
That's not a negative test, that's just an unsupported scenario that shouldn't be concerned with the driver.